lib/shared/config-manager/src/index.ts

```diff
@@ -17,12 +21,18 @@ export class EnvironmentConfigManager {
     private readonly requestTimeoutMS: number
     private readonly cdnURI: string
     fetchConfigPromise: Promise<void>
-    private intervalTimeout?: NodeJS.Timeout
+    private intervalTimeout?: any
     private disablePolling = false
+    private readonly setConfigBuffer: SetConfigBuffer
+    private readonly setInterval: SetIntervalInterface
+    private readonly clearInterval: ClearIntervalInterface
 
     constructor(
         logger: DVCLogger,
         sdkKey: string,
+        setConfigBuffer: SetConfigBuffer,
+        setInterval: SetIntervalInterface,
```

Comment:
this polling config fetcher seems very specific to the nodejs local bucketing SDK, what purpose would it serve in the worker SDK other than to fetch it once?

Reply:
workers do last for a while (I think I've seen 10's of minutes if not longer), we'd still want polling for them.